### PR TITLE
Uncomments condition that would allow Lamprey to be voted

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
If the station is a complete mess aka -20000 crew score or less then Lamprey will be unlocked
Requires @d3athrow to do stuff on his computer AFAIK, like Castle Station
:cl:
 * rscadd: Completely wrecking the station will now unlock the map you deserve.

### CODE REVIEW

**Comment by d3athrow** [comment]
lamprey is already compiled

**Comment by boy2mantwicethefam** [comment]
neat

**Comment by PrimeDSS13** [comment]
downvoted until you make it -20001 again

**Comment by GreatBigFailure** [comment]
Crew score for that sort of thing is fun

**Comment by SonixApache** [comment]
doesn't lamprey still kill the server

**Comment by boy2mantwicethefam** [comment]
it doesn't, it's been fixed a while ago

**Comment by SonixApache** [comment]
make it so lamprey voting can't fire from lamprey

**Comment by boy2mantwicethefam** [comment]
the ride never ends

File: code/__HELPERS/files.dm [hunk 1]
```diff
@@ -83,16 +83,13 @@
...
 				binary = null
 				continue
-//Lamprey currently has a tendency to crash for no real reason, uncomment this when it stops being broken
-/*
 		if(potential == "Lamprey Station/") //Available if the station is wrecked enough
-			var/crew_score = score["crewscore"] //So that we can use this in the chat
-			if(crew_score >= -20000)
+			var/crew_score = score["crewscore"] //So that we can use this in the admin messaging
+			if(crew_score > -20000)
 				message_admins("Skipping map [potential], stationrequires lower than -20000 score (is [score]).")
```
Comment (Kurfursten) [inline code/__HELPERS/files.dm]: you used [score] instead of [crew_score] or [score["crew_score"]]

**Comment by DeityLink** [comment]
I like it, that potentially makes Lamprey to cult rounds what Castlestation is to rev rounds.

File: code/__HELPERS/files.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/__HELPERS/files.dm]: Good catch

**Comment by mrflarg** [comment]
> I like it, that potentially makes Lamprey to cult rounds what Castlestation is to rev rounds.

Castle doesn't fire, we had a rev round recently and we got no option.

**Comment by Kurfursten** [comment]
That's because these Barry PRs are actually the opposite of what he says, they PREVENT the map from appearing on the mapvote if the condition is NOT met, it requires actually being added to the vote in configs after merge

**Comment by boy2mantwicethefam** [comment]
Biggest obstacle is that the files belong on Pomf's computer and the name of the files need to be exactly like in the folder
"Lamprey Station/"
"Castle Station/"

**Pull request merged**